README.md

```diff
@@ -1,8 +1,4 @@
-Daiquiri Application (Django version)
+Daiquiri Application (React Django version)
```

Comment:
Personally, I would leave out 'React Django version' or 'Django version' completely. It should be just Daiquiri. The link below undoubtedly shows which version of Daiquiri should be used. The PHP version is not maintained anymore.